taca/nanopore/minion.py

```diff
@@ -16,38 +16,50 @@ def __init__(self, run_dir, nanoseq_sample_sheet, anglerfish_sample_sheet):
         super(MinION, self).__init__(run_dir)
         self.nanoseq_sample_sheet = nanoseq_sample_sheet
         self.anglerfish_sample_sheet = anglerfish_sample_sheet
+               
         self.nanoseq_dir = os.path.join(self.run_dir, 'nanoseq_output')
         self.anglerfish_dir = os.path.join(self.run_dir, 'anglerfish_output')
         self.nanoseq_exit_status_file = os.path.join(self.run_dir, '.exitcode_for_nanoseq')
         self.anglerfish_exit_status_file = os.path.join(self.run_dir, '.exitcode_for_anglerfish')
+
         self.year_processed = self.run_id[0:4]
         self.flowcell_id = self.run_id.split('_')[3]
-
-    def parse_lims_sample_sheet(self):
-        """Generate nanoseq samplesheet based on Lims original."""
         self._get_original_samplesheet()
         if self.lims_samplesheet:
-            self._parse_samplesheet()
-        else:
-            self.nanoseq_sample_sheet = ''
+            self._set_run_type()
```

Comment:
Is Run type set only if a samplesheet exists?

Reply:
Yeah, it's based on the sample sheet name. If the sample sheet is generated by the QC step in lims, it will have a `QC` prefix, and if it's generated in the minion sequencing step it's prefixed with `DELIVERY`.

Reply:
The assumption is that a smaplesheet would always be generated, either in the QC step in LIMS or in the sequencing step.